src/main/java/Event.java

```diff
@@ -0,0 +1,24 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+// a task that need to specify exact date and time
+public class Event extends Task {
+
+    protected LocalDateTime at;
```

Comment:
According to the CS2103/T textbook, under the "Guideline: Name well" section in the Code Quality chapter, it says that nouns should be used for classes/variables. Maybe you could consider using a noun to describe your variable. You could maybe use `eventTime` or `dateTime` instead of  `at`.